config/config.go

```diff
@@ -0,0 +1,71 @@
+package config
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/yaml.v2"
+)
+
```

Comment:
add comment for exported structure and function

Reply:
Done!